software/youtubedl-server.yml

```
@@ -1,13 +1,13 @@
 name: YoutubeDL-Server
 website_url: https://github.com/nbr23/youtube-dl-server
-description: Web and REST interface for downloading videos onto a server.
+description: Youtube-DL interface for downloading videos onto a server.
```

```suggestion
description: Web and REST interface to Youtube-DL for downloading videos onto a server.
```
more accurate description